docs/concepts/vega-chain.md

```diff
@@ -85,61 +84,117 @@ Exceptions to automatic nomination:
 
-**Un-nominate towards the end of the epoch** 
-
+#### Un-nominate towards the end of the epoch
 A participant can un-nominate towards the end of the current epoch, which means the stake is not used for the validator from the following epoch. 
 
 The action is announced in the next available block of the same epoch, but the nominator keeps their nomination active until the last block of that epoch. At that point, the tokens are returned. The nominator cannot move those tokens before the epoch ends.
 
-**Un-nominate now**
-
+#### Un-nominate now
 A participant can choose to un-nominate at any time, and the action is executed immediately following the block it is announced in (within the same epoch). 
 
 The participant will not receive any rewards from the validator in that epoch. The tokens are marked as available to the participant.
 
-### Staking rewards
-Validators and nominators both receive incentives from the network, depending on factors including how much stake is nominated. 
+## Staking rewards & penalties
 
-**To be considered for staking rewards, a participant must associate VEGA to a Vega key and nominate one or more validators.**
+### Rewards
+Validators and nominators both receive incentives for securing the network. The amount of those incentives, rewarded as VEGA, depends on factors including how much stake is nominated. 
 
-For restricted mainnet, rewards are distributed among validators in proportion to their total stake. The total stake includes a validator's own stake and the tokens nominated to that validator. 
+**To be considered for staking rewards, a tokenholder must associate VEGA to a Vega key and nominate one or more validators.**
 
-At the end of each epoch, reward payments are calculated per active validator, and then some of that reward is divided between their nominators. The proportion that goes to nominators is defined by the network parameter `reward.staking.delegation.delegatorShare`. 
+In each epoch, rewards are distributed among validators in proportion to the number of tokens they represent (i.e., their total stake). The total stake includes a validator's own stake and the tokens nominated to that validator. Of this reward, a fixed amount is distributed among the tokenholders the validator represents. Currently that is 88.3%, though validators can vote to change that value.
 
-:::note Further reading
-Read the [staking rewards](https://github.com/vegaprotocol/specs/blob/main/protocol/0058-simple-POS-rewards.md) spec for full details for how rewards are calculated and will be in future iterations. 
-:::
+The reward scheme uses a linear reward curve - the reward per staked token is independent of the behaviour of other tokenholders. This holds for validators as well, with the exception that there is a maximum amount of stake an individual validator can take on. 
+
```

Comment:
However, to avoid validators getting too big, the reward a validator gets (and can distribute to its delegators) is capped, i.e., the reward per token is decreasing if a validator exceeds a maximum size.

Reply:
Added, thank you!